api/src/main/java/uk/gov/hmcts/payment/api/dto/BulkScanPaymentRequestStrategic.java

```diff
@@ -0,0 +1,104 @@
+package uk.gov.hmcts.payment.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Wither;
+import org.apache.commons.lang.StringUtils;
+import org.joda.time.DateTime;
+import uk.gov.hmcts.payment.api.contract.PaymentAllocationDto;
+import uk.gov.hmcts.payment.api.contract.util.CurrencyCode;
+import uk.gov.hmcts.payment.api.contract.util.Service;
+import uk.gov.hmcts.payment.api.model.PaymentChannel;
+import uk.gov.hmcts.payment.api.model.PaymentStatus;
+import uk.gov.hmcts.payment.api.util.PaymentMethodType;
+
+import javax.validation.constraints.*;
+import java.math.BigDecimal;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonInclude(NON_NULL)
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder(builderMethodName = "createBulkScanPaymentStrategicWith")
+public class BulkScanPaymentRequestStrategic {
+
```

Comment:
Also if we need this new POJO then I think we should have something better than 'BulkScanPaymentRequestStrategic'

Reply:
The strategic prefix added to identify new and old code of logic, which will simplify refactoring and removal of old endpoint related code in the future.

When we will be removing old endpoint code, we can refactor and remove this prefix as well

Reply:
I think better we can re-use the existing POJO BulkScanPaymentRequest in the new API, we can't code based on future assumptions.

Reply:
check above comment